.github/workflows/ci.yml

```
@@ -18,22 +18,23 @@ env:
   TOX_TESTENV_PASSENV: FORCE_COLOR
   PIP_DISABLE_PIP_VERSION_CHECK: "1"
   PIP_NO_PYTHON_VERSION_WARNING: "1"
-  # Keep in sync with .pre-commit-config.yaml/default_language_version/python.
-  PYTHON_LATEST: "3.11"
+  PYTHON_LATEST: "3.13"
 
 jobs:
   tests-sqlite:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12']
+        python-version: [PYTHON_LATEST]
```

It looks like this is not getting interpolated: https://github.com/torchbox/wagtail-grapple/actions/runs/13347176830/job/37279042624?pr=420#step:3:16.

```suggestion
        python-version: [$PYTHON_LATEST]
```

---

something, something.. copy/pasta fail